alias.config.js

```diff
@@ -2,18 +2,35 @@ const resolve = require('path').resolve;
 const fs = require('fs');
 
 const ROOT = resolve(__dirname, '.');
-const PACKAGES_ROOT = resolve(ROOT, 'src');
+const MODULES_ROOT = resolve(ROOT, 'modules');
 const ROOT_ALIAS = 'packages';
```

Comment:
This can be changed to `@mlvis` since you are planning to publish `@mlvis/feature-list-view` etc

Reply:
there quite some references to packages/mlvis-common still, I plan to fix them when we finish merging the latest PRs you made.